pypowsybl/network/impl/network.py

```diff
@@ -3939,17 +4007,18 @@ def create_batteries(self, df: DataFrame = None, **kwargs: ArrayLike) -> None:
         """
         return self._create_elements(ElementType.BATTERY, [df], **kwargs)
 
-    def create_dangling_lines(self, df: DataFrame = None, **kwargs: ArrayLike) -> None:
+    def create_dangling_lines(self, df: DataFrame = None, generation_df: DataFrame = pd.DataFrame(), **kwargs: ArrayLike) -> None:
```

Comment:
Is **kwargs stil working for example to pass the dangling base attributes inline and the generation part with generation_df ? 
```python
n.create_dangling_line(id='X', r=3.4, ..., generation_df=...)
```

Reply:
Yes, the tests use create_dangling_lines this way to verify it still works.
On the other end, data for generation_df cannot be passed using kwargs